api/tests/tooling/domain-builder/factory/build-competence-evaluation.js

```diff
@@ -20,7 +20,7 @@ module.exports = function buildCompetenceEvaluation(
     assessmentId = assessment.id;
   }
   if (assessmentId && !assessment) {
-    assessmentId = buildAssessment({ id: assessmentId });
+    assessmentId = buildAssessment({ id: assessmentId }).id;
```

Comment:
Houlà il est amusant cet enchaînement de `if` ! Il serait peut-être un peu plus clair avec des `else` pour les enchaîner, vu que chaque bloc modifie des variables qui servent dans les conditions des suivants.

J'ai l'impression qu'avec cette ligne modifiée on va toujours mettre dans `assessmentId`… la valeur de `assessmentId` donc ça serait équivalent à juste :
```js
    buildAssessment({ id: assessmentId });
```

Mais si je regarde un peu plus le contexte, il me semble que l'intention du groupe de `if` est de s'assurer que `assessment` et `assessmentId` soient toujours définis quelles que soient leurs valeurs à l'entrée. Du coup il y avait bien une erreur mais c'est plutôt dans le nom de la variable à assigner :
```suggestion
    assessment = buildAssessment({ id: assessmentId });
```